backwards-compatability/node10/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "@opentelemetry/sdk-node": "0.25.0",
```

Comment:
This seems to be an unwanted dependency to an old version.

Reply:
why ? the sdk node is experimental so its still in 0.25.0 ?

Reply:
We may have to duplicate the backwards-compatibility test into the experimental folder. I realized it isn't linking against the experimental packages now so isn't providing value to them.

Reply:
`@opentelemetry/sdk-node` depends on all non experimental modules in version 0.25.0. As a result we have two versions of the modules installed and I doubt this is what we want.

Reply:
Do you mean in the backwards-compatibility test? sdk-node should probably be removed from this particular test since it isn't really being tested

Reply:
```js
import {NodeSDK, api} from '@opentelemetry/sdk-node';
import {ConsoleSpanExporter} from '@opentelemetry/sdk-trace-base';
```
This test instantiates the NodeSDK 0.25.0 and uses the `ConsoleSpanExporter` from 0.26.0.

Reply:
Yeah that probably means this test should be moved to the experimental folder. It is doing basically nothing here.

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/2508